Admin/StaticContentAdmin.php

```diff
@@ -48,20 +46,27 @@ protected function configureListFields(ListMapper $listMapper)
 
     protected function configureFormFields(FormMapper $formMapper)
     {
+        $formFactory = $formMapper->getFormBuilder()->getFormFactory();
+        $builder = $formFactory->createBuilder(StaticContentType::class, null, [
+            'use_ivory_ckeditor' => !empty($this->ivoryCkeditor),
+            'ivory_ckeditor_options' => $this->ivoryCkeditor,
+        ]);
+
+        // Choose the type for parentDocument field. TreeSelectType if creating the document, TextType when editing
+        $parent = $builder->get('parentDocument');
+        if ($this->id($this->getSubject())) {
+            $parent = $formFactory->createNamedBuilder('parentDocument', TextType::class, null, [
+                'disabled' => true,
+            ]);
+            $parent->addModelTransformer(new DocumentToPathTransformer($this->getModelManager()->getDocumentManager()));
```

Comment:
this always has to be applied, also when creating a new document (then `reverseTransform` is called on form submit)